x-pack/platform/plugins/shared/ml/server/models/data_recognizer/modules/security_gcp_audit/ml/gcp_audit_high_distinct_count_error_message.json

```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Security: GCP Audit - Looks for a spike in the rate of an actions when the event outcome is a failure, which may simply indicate an impending service failure but these can also be byproducts of attempted or successful persistence, privilege escalation, defense evasion, discovery, lateral movement, or collection activity by a threat actor.",
+  "groups": ["security", "gcp"],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "Detects high distinct count of failing event actions.",
+        "function": "high_distinct_count",
```

Comment:
To make sure I understand, does `status.message` guarantee that it's related to failures? Seeing the description "spike in the rate of an actions when the event outcome is a failure" (I could have missed it)
Same comment for the Azure equivalent of this job